pkg/kv/migrations/migrations_test.go

```diff
@@ -13,7 +13,6 @@ import (
 	authacl "github.com/treeverse/lakefs/contrib/auth/acl"
```

Comment:
Can we open an issue to get rid of these migrations?  Not sure we need them any longer, and after we clean up auth / ACL I would prefer to lose this code.

Reply:
https://github.com/treeverse/lakeFS/issues/8077 FYI @guy-har